ember-composable-helpers/package.json

```diff
@@ -21,12 +26,24 @@
     "@babel/core": "^7.0.0",
     "@babel/plugin-proposal-decorators": "^7.22.15",
     "@babel/plugin-transform-class-properties": "^7.22.5",
+    "@babel/plugin-transform-typescript": "^7.23.6",
     "@babel/runtime": "^7.22.15",
     "@embroider/addon-dev": "^4.1.0",
+    "@glint/core": "^1.2.1",
+    "@glint/environment-ember-loose": "^1.2.1",
+    "@glint/environment-ember-template-imports": "^1.2.1",
+    "@glint/template": "^1.2.1",
     "@rollup/plugin-babel": "^6.0.3",
+    "@tsconfig/ember": "^3.0.2",
+    "@types/rsvp": "^4.0.9",
+    "@typescript-eslint/eslint-plugin": "^6.14.0",
+    "@typescript-eslint/parser": "^6.14.0",
     "concurrently": "^8.2.1",
+    "ember-source": "~5.8.0",
```

Comment:
Needed to install this + `webpack` peer to access built-in types

Reply:
makes sense